portfolio/src/main/webapp/about.html

```diff
@@ -17,7 +18,7 @@ <h1>About</h1>
   <div class="menu">
     <nav>
         <a href="index.html"> Home </a>
-	<a href="gallery.html"> Photos </a>
+	      <a href="gallery.html"> Photos </a>
```

Comment:
I think Cloud Code is set to 4 spaces vs. preset code's 2 spaces.
When you work on the capstone, you get to decide on your own code style as a pod!